{
    "$ref": "https://www.cert.ssi.gouv.fr/openapi.json",
    "affected_systems": [
        {
            "description": "CPython sans le dernier correctif de sécurité",
            "product": {
                "name": "CPython",
                "vendor": {
                    "name": "Python",
                    "scada": false
                }
            }
        }
    ],
    "affected_systems_content": "",
    "content": "## Solutions\n\nSe référer au bulletin de sécurité de l'éditeur pour l'obtention des correctifs (cf. section Documentation).",
    "cves": [
        {
            "name": "CVE-2025-4516",
            "url": "https://www.cve.org/CVERecord?id=CVE-2025-4516"
        }
    ],
    "links": [],
    "reference": "CERTFR-2025-AVI-0415",
    "revisions": [
        {
            "description": "Version initiale",
            "revision_date": "2025-05-16T00:00:00.000000"
        }
    ],
    "risks": [
        {
            "description": "Déni de service"
        }
    ],
    "summary": "Une vulnérabilité a été découverte dans Python. Elle permet à un attaquant de provoquer un déni de service.",
    "title": "Vulnérabilité dans Python",
    "vendor_advisories": [
        {
            "published_at": "2025-05-15",
            "title": "Bulletin de sécurité Python L75IPBBTSCYEF56I2M4KIW353BB3AY74",
            "url": "https://mail.python.org/archives/list/security-announce@python.org/thread/L75IPBBTSCYEF56I2M4KIW353BB3AY74/"
        }
    ]
}